src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -58,5 +60,6 @@ public PersonCard(Person person, int displayedIndex) {
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        history.setText("history placeholder");
```

Comment:
Magic string and add test

Reply:
This is a placeholder and need to be replaced in relevant ui change